src/System.Runtime.CompilerServices.VisualC/System.Runtime.CompilerServices.VisualC.sln

```diff
@@ -13,26 +13,63 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{E107E9C1-E89
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "ref", "ref", "{2E666815-2EDB-464B-9DF6-380BF4789AD4}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "tests", "tests", "{D6B41179-95FE-4ACA-8A26-D9C743A54E11}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "System.Runtime.CompilerServices.VisualC.Tests", "tests\System.Runtime.CompilerServices.VisualC.Tests.csproj", "{8012DD70-A6D7-45C0-BC8E-DFFB48D86E08}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
+		netcoreapp-Debug|Any CPU = netcoreapp-Debug|Any CPU
+		netcoreapp-Release|Any CPU = netcoreapp-Release|Any CPU
 		Release|Any CPU = Release|Any CPU
+		uap-Debug|Any CPU = uap-Debug|Any CPU
+		uap-Release|Any CPU = uap-Release|Any CPU
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
 		{F8C8FBFC-CA6E-4BDA-A1B3-FA4E17C10D9E}.Debug|Any CPU.ActiveCfg = netcoreapp-Debug|Any CPU
 		{F8C8FBFC-CA6E-4BDA-A1B3-FA4E17C10D9E}.Debug|Any CPU.Build.0 = netcoreapp-Debug|Any CPU
+		{F8C8FBFC-CA6E-4BDA-A1B3-FA4E17C10D9E}.netcoreapp-Debug|Any CPU.ActiveCfg = netcoreapp-Debug|Any CPU
```

Comment:
Did you run the UpdateVSConfigurations task? I wouldn't expect to see all these configurations in the solution file.